docs/errors.md

```diff
@@ -304,3 +304,41 @@ Something went wrong with your custom ReactNodeView Component.
 > I18N Context
 
 There is something wrong with your i18n setup.
+
+### RMR0301
```

Comment:
Could you change the title to the following? It's not actually possible for the **remirror* runtime to capture the custom error `RMR0301` since it happens during the build process. 

```suggestion
## Build Errors

### `make-plural/plurals`
```